### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by PavlovaOlena**
[DEMO LINK](https://PavlovaOlena.github.io/layout_html-form/)
[TEST REPORT LINK](https://PavlovaOlena.github.io/layout_html-form/report/html_report/)

### CODE REVIEW

**Review by mykhalenych — CHANGES_REQUESTED** [review]
your link to tests isn't working

**Comment by PavlovaOlena** [comment]
> your link to tests isn't working

@mykhalenych I fixed test report

File: src/index.html [hunk 1]
```diff
@@ -1,17 +1,142 @@
...
+      class="form"
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+      method="post"
+      name="registrationForm"
+    >
+      <fieldset class="form-section personalInformation">
+        <legend class="title-form-section">Personal information</legend>
+        <label class="form-field">
+          Surname:
+          <input type="text" name="surname" autocomplete="off">
```
Comment (YegorKorenkov) [inline src/index.html]: Please read checklist and fix for all cases 
image

File: src/index.html [hunk 2]
```diff
@@ -1,17 +1,142 @@
...
+            maxlength="30"
+          >
+        </label>
+      </fieldset>
+
+      <fieldset class="form-section interestingFacts">
+        <legend class="title-form-section">An interesting fact about you!</legend>
+        <div class="form-field">
+          Do you love cats?
+          <label for="loveCats">
```
Comment (YegorKorenkov) [inline src/index.html]: Is I click on label 'Yes', input radio 'Yes' not selected, please fix
image

File: src/index.html [hunk 3]
```diff
@@ -1,17 +1,142 @@
...
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form
+      class="form"
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+      method="post"
+      name="registrationForm"
+    >
+      <fieldset class="form-section personalInformation">
```
Comment (YegorKorenkov) [inline src/index.html]: Why you need personalInformation class? You don't need to specify it if you don't want to apply any styles. Also fix for all cases!!!

**Review by YegorKorenkov — COMMENTED** [review]
Please see comments. Also please read and make sure you complete all tasks

File: src/index.html [hunk 4]
```diff
@@ -1,17 +1,142 @@
...
+        <legend class="title-form-section">Personal information</legend>
+        <label class="form-field">
+          Surname:
+          <input type="text" name="surname" autocomplete="off">
+        </label>
+        <label class="form-field">
```
Comment (Anastasiia-Svintsova) [inline src/index.html]: Don't forget to add empty lines between multiline sibling blocks of HTML. Check it everywhere, please 
```suggestion
        </label>
        
        <label class="form-field">
```

File: src/index.html (hunk 1 above)
Comment (Anastasiia-Svintsova) [inline src/index.html]: Check it everywhere, please 
image
```suggestion
          <input 
            type="text" 
            name="surname" 
            autocomplete="off"
          >
```

**Review by Anastasiia-Svintsova — CHANGES_REQUESTED** [review]
- You should check the label for the `Yes` button

https://user-images.githubusercontent.com/99403198/221537920-6915e6fa-d850-4921-a723-53543ffbc165.mov

- Options should start from the top. Check the mockup
image

- You should improve the code readability, pay attention to my comments, please

File: src/index.html [hunk 5]
```diff
@@ -1,17 +1,174 @@
...
+            multiple
+          >
+            <option 
+              value=" " 
+              disabled 
+              selected
+            ></option>
+            <option value="BMW">BMW</option>
+            <option value="Audi">Audi</option>
+            <option value="Lada">Lada</option>
```
Comment (anstsot) [inline src/index.html]: options should start from the top as was mentioned before. let's move the first option to the bottom
image
```suggestion
            <option value="BMW">BMW</option>
            <option value="Audi">Audi</option>
            <option value="Lada">Lada</option>
            <option 
              value=" " 
              disabled 
              selected
            ></option>
```

**Review by anstsot — APPROVED** [review]
Good job, I'm approving! But it would be great if you fix one comment

File: src/index.html (hunk 5 above)
Comment (PavlovaOlena) [reply src/index.html]: Thank you ))

**Pull request closed without merging**